library/ix-dev/community/netbootxyz/templates/_service.tpl

```diff
@@ -0,0 +1,32 @@
+{{- define "netboot.service" -}}
+service:
+  netboot:
+    enabled: true
+    primary: true
+    type: NodePort
+    targetSelector: netboot
+    ports:
+      tftp:
+        enabled: true
+        primary: true
+        port: {{ .Values.netbootNetwork.tftpPort }}
+        nodePort: {{ .Values.netbootNetwork.tftpPort }}
```

Comment:
Unless there is a variable to also adjust the port inside the container,
You will have to also set `targetPort` to internal port